index.html

```diff
@@ -32,5 +32,159 @@ <h3>LET'S CONNECT</h3>
                 </ul>
             </section>
         </div>
+        <section class="work">
+            <div class="projects">
+                <img class="snapshots" id="snap1" src="snap.svg">
+                <div class="info_block">
+                    <div class="projo">
+                        <h4 class="project_name">Tonic</h4>
+                        <ul class="deatils">
+                            <li class="detail">CANOPY</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">Back End Dev</li>
+                            <li class="detail"><img src="Counter.svg"></li>
+                            <li class="detail">2023</li>
+                        </ul>
+                    </div>
+                    <p class="info">A daily selection of privately personalized reads; no accounts
```

Comment:
- [ ] I would kindly suggest you to document your project very well by arrange all images inside one image folder to avoid broken images on your page,kindly fix the broken images not display as the original design .
- [ ] your design
![Screenshot 2023-06-03 034926](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/b906658f-d4ec-4d11-a3dd-42b698545e6a)
 
figma design
![works-mobile](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/92458236/aa6c132c-85cb-41cd-9a01-c8fc87616168)